tests/sentry/hybridcloud/test_caching.py

```diff
@@ -136,15 +137,15 @@ def reader():
                 yield from CacheBackend.set_cache(shared_key, copied_local_value, version)
                 last_length = len(copied_local_value)
 
-    def writer():
+    def writer() -> Iterator[None]:
```

Comment:
I would be more specific and use `Generator[None, None, None]` though it doesn't matter here I don't think